LongRunning/src/LongRunning/OperationsClient.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * GENERATED CODE WARNING
+ * Generated by gapic-generator-php from the file
+ * https://github.com/googleapis/googleapis/blob/master/google/longrunning/operations.proto
+ * Updates to the above are reflected here through a refresh process.
+ */
+
+namespace Google\LongRunning;
+
+use Google\LongRunning\Gapic\OperationsGapicClient;
+
+/** {@inheritdoc} */
+class OperationsClient extends OperationsGapicClient
+{
+    // This class is intentionally empty, and is intended to hold manual additions to
+    // the generated {@see OperationsGapicClient} class.
+}
+
+class_alias('Google\LongRunning\OperationsClient', 'Google\ApiCore\LongRunning\OperationsClient');
```

Comment:
Why do we need an Alias? My guess is that we are trying to avoid breaking changes. Is that right?

Reply:
yes that's correct! We don't want to introduce any behavioral change